src/components/proofs/ReadyERC721ProofsList.tsx

```diff
@@ -1,25 +1,22 @@
 import { useSnapshot } from 'valtio'
 import ContractListContainer from 'components/proofs/ContractListContainer'
+import ERC721Proof from 'helpers/proofs/ERC721Proof'
 import Network from 'models/Network'
 import Proof from 'components/proofs/Proof'
 import ProofStore from 'stores/ProofStore'
-import networkPick from 'helpers/networkPick'
 
 export default function ({ network }: { network: Network }) {
-  const { goerliERC721ProofsCompleted, mainnetERC721ProofsCompleted } =
-    useSnapshot(ProofStore)
+  const { proofsCompleted } = useSnapshot(ProofStore)
 
-  const ERC721ProofsCompleted = networkPick(
-    network,
-    goerliERC721ProofsCompleted,
-    mainnetERC721ProofsCompleted
-  )
+  const filtered = proofsCompleted.filter(
+    (proof) => proof instanceof ERC721Proof && proof.network === network
+  ) as ERC721Proof[]
```

Comment:
Hm, do you need to convert it? You already checked for `instanceof` 🤔

Reply:
And let's name this `const` properly, I know that you filtered something, but what is the result of this filtering? I can read the code to understand, but isn't it's better to just read the name of the variable and understand it faster?